codegen/masm/src/emulator/mod.rs

```diff
@@ -572,7 +572,7 @@ impl Emulator {
     pub fn stop(&mut self) {
         self.callstack.clear();
         self.stack.clear();
-        self.memory.clear();
+        self.memory = vec![Self::EMPTY_WORD; self.memory.len()];
```

Comment:
```suggestion
        self.memory.fill(Self::EMPTY_WORD);
```

Reply:
Good catch! In #237, I reworked it completely (it only cleans "dirty" addresses).